docs/zkapps/o1js/basic-concepts.md

```diff
@@ -127,25 +126,106 @@ You can think assertions in o1js similar to `require()` function in Solidity.
 
 ## Printing a Field to the Screen
 
-`Field` elements are stored as classes in o1js. This is why if you print a `Field` without a conversion, you will get an unreadable output. Hopefully, o1js gives a variety of conversion methods to print `Field` elements to the screen.
+`Field` elements are stored as classes in o1js. This is why if you print a `Field` using `console.log`, you will get an unreadable output. Hopefully, o1js provides an equivalent function to `console.log` for provable types: `Provable.log`
 
 <!-- prettier-ignore -->
 ```ts
-const x = Field(45);
+let x = Field(45);
+Provable.log(x); // Prints 45
+```
+
+## Other Types and Methods
+
+In addition to the `Field` type, o1js gives you a variety structures to help you build a powerful zkApp. You can learn more about them in the next chapter, [Common Types and Functions](./common-types-and-functions.md).
+
+:::note
+
+The information given until here is more than enough for you to start creating powerful zkApps with o1js. For more advanced level content, you can read the following sections. If you are new to o1js, just continue with the next chapter, [Common Types and Functions](./common-types-and-functions.md).
+
+:::
+
+## Deeper into the Field Type
+
+`Field` element is a positive integer in a prime order [finite field](https://en.wikipedia.org/wiki/Finite_field), which is a set of integers in between 1 and the prime order of the field (a.k.a. an arbitrary prime number describing the field).
```

Comment:
Actually it's the set of integers between 0 and p-1, where p is the prime order!

Reply:
```suggestion
A `Field` represents a positive integer in a prime order [finite field](https://en.wikipedia.org/wiki/Finite_field), which is the set of integers $0,...,p-1$, where $p$ is the field order.

Concretely, `Field` refers to the [Pallas base field](TODO), which is the native circuit field of the proof system underlying o1js.
```

@yunus433 this is how I would describe the field